force-app/main/default/lwc/givingSummary/givingSummary.js

```diff
@@ -0,0 +1,86 @@
+import { api, LightningElement, wire } from 'lwc';
+import { getRecord } from 'lightning/uiRecordApi';
+import donorsGivingSummary from '@salesforce/label/c.donorsGivingSummary';
+import donorsLifetime from '@salesforce/label/c.donorsLifetime';
+import donorsThisYear from '@salesforce/label/c.donorsThisYear';
+import donorsPreviousYear from '@salesforce/label/c.donorsPreviousYear';
+import TOTAL_AMOUNT from '@salesforce/schema/Contact.npo02__TotalOppAmount__c';
+import AMOUNT_CURRENT_YEAR from '@salesforce/schema/Contact.npo02__OppAmountThisYear__c';
+import AMOUNT_LAST_YEAR from '@salesforce/schema/Contact.npo02__OppAmountLastYear__c';
+import FORM_FACTOR from '@salesforce/client/formFactor';
+
+const FormFactorType = Object.freeze({
+    Large: 'Large',
+    Medium: 'Medium',
+    Small: 'Small',
+});
+
+const MOBILE_CLASSES = 'slds-text-align_center slds-border_bottom slds-text-heading_small slds-var-p-around_medium';
+const DESKTOP_CLASSES = 'slds-text-align_center slds-border_right slds-text-heading_small';
+const FIELDS = [TOTAL_AMOUNT, AMOUNT_CURRENT_YEAR, AMOUNT_LAST_YEAR];
+
+export default class GivingSummary extends LightningElement {
+    labels = {
+        donorsGivingSummary,
+        donorsLifetime,
+        donorsThisYear,
+        donorsPreviousYear,
+    };
+
+    lifetimeSummary = 0;
+    thisYear = 0;
+    previousYear = 0;
+    
+    contact;
+    @api contactId;
+
+    message;
+    
+    formFactor = FORM_FACTOR;
+
+    @wire(getRecord, { recordId: '$contactId', fields: FIELDS})
+        wiredRecord({ error, data }) {
+            if (error) {
+                    this.message = 'Unknown error';
```

Comment:
Let's replace this with the `commonUnknownError` label.

Reply:
done